### PR TITLE
Feature/2046 splash ui

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #2046 

## Relevant technical choices

Conditionally set layout specific class names if in development mode to show the updated UI with the image.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/setup/setup-proxy.js [hunk 1]
```diff
@@ -75,28 +77,24 @@ function SetupUsingProxy() {
 
 	let title;
 	let description;
-	let startSetupText;
+	const startSetupText = __( 'Sign in with Google', 'google-site-kit' );
```
Comment (eugene-manuilov) [inline assets/js/components/setup/setup-proxy.js:80]: Let's move it to where it's used in the markup.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
LGTM, there is just one thing to improve:

File: assets/js/components/setup/setup-proxy.js (hunk 1 above)
Comment (asvinb) [reply assets/js/components/setup/setup-proxy.js]: Updated @eugene-manuilov

**Review by eugene-manuilov — APPROVED** [review]

**Comment by asvinb** [comment]
@eugene-manuilov Just FYI, VR tests failing because of an image not loading.

File: assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss [hunk 2]
```diff
@@ -37,4 +37,28 @@
 			padding: $grid-gap-desktop 0 0 0;
 		}
 	}
+
+	.googlesitekit-wizard-progress__content {
+		margin: 0 auto;
+		max-width: 465px;
```
Comment (felixarntz) [inline assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss:43]: Just curious, why did you choose this maximum width specifically? Feels a bit like an arbitrary value, maybe this should be in a variable somewhere?

File: assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss [hunk 3]
```diff
@@ -37,4 +37,28 @@
 			padding: $grid-gap-desktop 0 0 0;
 		}
 	}
```
Comment (felixarntz) [inline assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss:39]: The rules in this file almost all apply to the legacy setup flow which is only visible when _not_ using the proxy (only relevant for development or staging sites). The `SetupProxy` component only uses `.googlesitekit-wizard-progress`, which is some legacy remainder. We don't need to change that at this point, but let's rather decouple the new classes from that and make them based on `.googlesitekit-setup__*` (and put them into `_googlesitekit-setup.scss`).

**Review by felixarntz — CHANGES_REQUESTED** [review]
Mostly LGTM, just 2 comments and one other thing: The margin above and below the opt-in checkbox is a bit small and looks odd, quite different from the mocks. Can you make that margin be the same `1rem` like what is around the description?

File: assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss (hunk 2 above)
Comment (asvinb) [reply assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss]: @felixarntz So the max width is as per the provided designs. If you look at the dashboard setup here: https://www.figma.com/file/f07vatCPHzubuJoXkgd5RU/SiteKit-Phase-1_-Onboarding?node-id=1098%3A17677 , the content is centered with a max width of 465px. I replaced it by a variable though. Let me know if that works for you or do you think we should get rid of the max width or shall we follow the design closely?

File: assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss (hunk 3 above)
Comment (asvinb) [reply assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss]: Sounds good @felixarntz . I moved the new class names. Can you check again?

**Comment by asvinb** [comment]
> Mostly LGTM, just 2 comments and one other thing: The margin above and below the opt-in checkbox is a bit small and looks odd, quite different from the mocks. Can you make that margin be the same `1rem` like what is around the description?

I added some more breathing room around the opt-in checkbox. Let me know if it's ok now. I specifically added the margin around the opt-in checkbox only if we are showing the person icon since I wanted to leave everything as is if `featureFlags.widgets.userInput.enabled` is false.

**Review by felixarntz — APPROVED** [review]
Awesome!

**Pull request merged**